internal/filter/regexTitle.go

```diff
@@ -0,0 +1,40 @@
+package filter
+
+import (
+	"regexp"
+
+	"github.com/charmbracelet/log"
+
+	"github.com/inovex/CalendarSync/internal/models"
+)
+
+type RegexTitle struct {
+	Regexp string
```

Comment:
As I just got confused whether the regexp "keeps" or "excludes" entries, what do you think about renaming the parameter to "ExcludeRegexp"?

Reply:
you're right. Renamed it to "ExcludeRegexp".